benchmarks/README.md

```diff
@@ -136,10 +136,10 @@ Benchmark run can be started with
           --model kNN \
           --budget 80 \
           --repeat 10 \
-          --sampler-list TPESampler CmaEsSampler \
-          --sampler-kwargs-list {"multivariate":true,"constant_liar":true} {} \
-          --pruner-list NopPruner \
-          --pruner-kwargs-list {}
+          --sampler-list "TPESampler CmaEsSampler" \
+          --sampler-kwargs-list "{\"multivariate\":true\,\"constant_liar\":true} {}" \
```

Comment:
Does this code work on your environment? I'm concerning `\,` causes JSON decode error. 🤔